storage-resize-images/functions/src/resize-image.ts

```diff
@@ -96,24 +99,15 @@ export const modifyImage = async ({
   contentType: string;
   size: string;
   objectMetadata: ObjectMetadata;
+  format: string;
 }): Promise<ResizedImageResult> => {
-  const { imageType } = config;
-  const hasImageTypeConfigSet = imageType !== "false";
-  const imageContentType = hasImageTypeConfigSet
-    ? supportedImageContentTypeMap[imageType]
+  const useOriginalFormat = format !== "raw";
+  const imageContentType = useOriginalFormat
+    ? supportedImageContentTypeMap[format]
     : contentType;
   const modifiedExtensionName =
-    fileExtension && hasImageTypeConfigSet ? `.${imageType}` : fileExtension;
-
-  let modifiedFileName;
-
-  if (supportedExtensions.includes(fileExtension)) {
-    modifiedFileName = `${fileNameWithoutExtension}_${size}${modifiedExtensionName}`;
-  } else {
-    // Fixes https://github.com/firebase/extensions/issues/476
-    modifiedFileName = `${fileNameWithoutExtension}${fileExtension}_${size}`;
-  }
```

Comment:
put this code back in, please. It was implemented for this [fix](https://github.com/firebase/extensions/issues/476)

Reply:
No problem. Looks like it was removed in error, adding back in...

Reply:
Code re-added.